development.ini.example

```diff
@@ -102,6 +102,10 @@ badge_ids = binary-star|both-bull-and-self-transcended-tester-viii|catching-the-
 # comps_dir = %(here)s/masher/comps
 # comps_url = https://git.fedorahosted.org/comps.git
 
+## Pungi configuration files
+# pungi_conf_dir = /etc/bodhi/pungi/
```

Comment:
Let's add this and its default to ```bodhi.server.config.BodhiConfig._defaults```.

Reply:
FYI, I added a default, but I also changed this variable to `pungi_modular_conf_filename`.

Reply:
https://github.com/mcurlej/bodhi/pull/20